src/map.h

```diff
@@ -666,19 +667,4 @@ static inline bool IsVoxelInsideWorld(const XYZPoint16 &vox)
 	return vox.z >= 0 && vox.z < WORLD_Z_SIZE && IsVoxelstackInsideWorld(vox.x, vox.y);
 }
 
-/**
- * Is both corners of given edge raised of a voxel with the
- * given non-steep slope?
- * @param edge The edge to check
- * @param slope The slope of a voxel.
- * @pre (slope & TSB_STEEP) == 0
- * @return True if both corners of given edge are raised, otherwise false.
- */
-static inline bool IsRaisedEdge(TileEdge edge, TileSlope slope)
```

Comment:
It was used with the old fence builder code, as fences at a raised edge are in the voxel above the base level. That use got killed with new GetVoxelZOffsetForFence  function in commit 64a4f50 which generalizes the query. Currently I wouldn't know anything else that needs to know about raised edges. Paths need flat edges (ie both raised or both down), but that is more generic than this function. Most other code mostly cares about ground as "not be where I want to be" without caring about actual shape of the ground.